AppCenter/AppCenter/Internals/Sender/MSHttpSenderPrivate.h

```diff
@@ -8,6 +8,12 @@
 
 @property(nonatomic) NSURLSession *session;
 
+/**
+ * The maximum number of connections for the session. The one collector endpoint only
+ * allows for two connections.
```

Comment:
It's contradictory with the value (4) passed at the init for the max connections.

Reply:
I'll update the comment so it reflects the two values we pass in.